2023-09-25 19:51:06 [http-nio-auto-1-exec-2] ERROR o.h.e.jdbc.spi.SqlExceptionHelper -
                Table "orders" not found; SQL statement:
select o1_0."order_pk",o1_0."closed",o1_0."created",o1_0."customer_fk",o1_0."order_total",o1_0."order_uid",o1_0."ready" from "orders" o1_0 [42102-214]
2023-09-25 19:57:46 [http-nio-auto-1-exec-2] ERROR o.h.e.jdbc.spi.SqlExceptionHelper -
                Table "orders" not found; SQL statement:
select o1_0."order_pk",o1_0."closed",o1_0."created",o1_0."customer_fk",o1_0."order_total",o1_0."order_uid",o1_0."ready" from "orders" o1_0 [42102-214]
2023-09-25 19:59:17 [main] ERROR o.h.e.jdbc.spi.SqlExceptionHelper -
                Table "orders" not found; SQL statement:
select o1_0."order_pk",o1_0."closed",o1_0."created",o1_0."customer_fk",o1_0."order_total",o1_0."order_uid",o1_0."ready" from "orders" o1_0 [42102-214]
2023-09-25 20:01:08 [main] ERROR o.h.e.jdbc.spi.SqlExceptionHelper -
                Table "orders" not found; SQL statement:
select o1_0."order_pk",o1_0."closed",o1_0."created",o1_0."customer_fk",o1_0."order_total",o1_0."order_uid",o1_0."ready" from "orders" o1_0 [42102-214]
2023-09-25 20:03:46 [main] ERROR o.h.e.jdbc.spi.SqlExceptionHelper -
                Table "orders" not found; SQL statement:
select o1_0."order_pk",o1_0."closed",o1_0."created",o1_0."customer_fk",o1_0."order_total",o1_0."order_uid",o1_0."ready" from "orders" o1_0 [42102-214]
2023-09-26 12:27:23 [http-nio-auto-1-exec-10] ERROR o.h.e.jdbc.spi.SqlExceptionHelper -
                NULL not allowed for column "customer_uid"; SQL statement:
insert into "customer" ("customer_pk", "customer_uid", "email", "name", "phone_number") values (default, ?, ?, ?, ?) [23502-214]
2023-09-26 12:27:24 [http-nio-auto-1-exec-2] ERROR o.h.e.jdbc.spi.SqlExceptionHelper -
                NULL not allowed for column "customer_uid"; SQL statement:
insert into "customer" ("customer_pk", "customer_uid", "email", "name", "phone_number") values (default, ?, ?, ?, ?) [23502-214]
2023-09-26 12:29:56 [http-nio-auto-1-exec-1] ERROR o.h.e.jdbc.spi.SqlExceptionHelper -
                NULL not allowed for column "customer_uid"; SQL statement:
insert into "customer" ("customer_pk", "customer_uid", "email", "name", "phone_number") values (default, ?, ?, ?, ?) [23502-214]
2023-09-26 12:29:56 [http-nio-auto-1-exec-3] ERROR o.h.e.jdbc.spi.SqlExceptionHelper -
                NULL not allowed for column "customer_uid"; SQL statement:
insert into "customer" ("customer_pk", "customer_uid", "email", "name", "phone_number") values (default, ?, ?, ?, ?) [23502-214]
2023-09-26 12:31:00 [http-nio-auto-1-exec-2] ERROR o.h.e.jdbc.spi.SqlExceptionHelper -
                NULL not allowed for column "customer_uid"; SQL statement:
insert into "customer" ("customer_pk", "customer_uid", "email", "name", "phone_number") values (default, ?, ?, ?, ?) [23502-214]
2023-09-26 14:17:18 [http-nio-auto-1-exec-2] ERROR o.h.e.jdbc.spi.SqlExceptionHelper -
                NULL not allowed for column "customer_uid"; SQL statement:
insert into "customer" ("customer_pk", "customer_uid", "email", "name", "phone_number") values (default, ?, ?, ?, ?) [23502-214]
2023-09-26 14:20:01 [http-nio-auto-1-exec-2] ERROR o.h.e.jdbc.spi.SqlExceptionHelper -
                NULL not allowed for column "customer_uid"; SQL statement:
insert into "customer" ("customer_pk", "customer_uid", "email", "name", "phone_number") values (default, ?, ?, ?, ?) [23502-214]
2023-09-26 15:02:46 [http-nio-auto-1-exec-2] ERROR o.h.e.jdbc.spi.SqlExceptionHelper -
                NULL not allowed for column "customer_uid"; SQL statement:
insert into "customer" ("customer_pk", "customer_uid", "email", "name", "phone_number") values (default, ?, ?, ?, ?) [23502-214]
2023-09-26 15:14:44 [http-nio-auto-1-exec-2] ERROR o.h.e.jdbc.spi.SqlExceptionHelper -
                Column "o1_0.customer_uid" not found; SQL statement:
select o1_0."order_pk",o1_0."closed",o1_0."created",o1_0."customer_fk",o1_0."customer_uid",o1_0."order_total",o1_0."order_uid",o1_0."ready" from "orders" o1_0 where o1_0."order_uid"=? [42122-214]
2023-09-26 15:19:55 [http-nio-auto-1-exec-2] ERROR o.h.e.jdbc.spi.SqlExceptionHelper -
                Column "o1_0.customer_uid" not found; SQL statement:
select o1_0."order_pk",o1_0."closed",o1_0."created",o1_0."customer_fk",o1_0."customer_uid",o1_0."order_total",o1_0."order_uid",o1_0."ready" from "orders" o1_0 where o1_0."order_uid"=? [42122-214]
2023-09-26 15:22:33 [http-nio-auto-1-exec-2] ERROR o.h.e.jdbc.spi.SqlExceptionHelper -
                Column "o1_0.customer_uid" not found; SQL statement:
select o1_0."order_pk",o1_0."closed",o1_0."created",o1_0."customer_fk",o1_0."customer_uid",o1_0."order_total",o1_0."order_uid",o1_0."ready" from "orders" o1_0 where o1_0."order_uid"=? [42122-214]
2023-09-26 15:23:42 [http-nio-auto-1-exec-2] ERROR o.h.e.jdbc.spi.SqlExceptionHelper -
                Column "o1_0.customer_uid" not found; SQL statement:
select o1_0."order_pk",o1_0."closed",o1_0."created",o1_0."customer_fk",o1_0."customer_uid",o1_0."order_total",o1_0."order_uid",o1_0."ready" from "orders" o1_0 where o1_0."order_uid"=? [42122-214]
2023-09-26 15:29:29 [http-nio-auto-1-exec-3] ERROR o.h.e.jdbc.spi.SqlExceptionHelper -
                Column "o1_0.customer_uid" not found; SQL statement:
select o1_0."order_pk",o1_0."closed",o1_0."created",o1_0."customer_fk",o1_0."customer_uid",o1_0."order_total",o1_0."order_uid",o1_0."ready" from "orders" o1_0 where o1_0."order_uid"=? [42122-214]
2023-09-26 15:31:15 [http-nio-auto-1-exec-2] ERROR o.h.e.jdbc.spi.SqlExceptionHelper -
                Column "o1_0.customer_uid" not found; SQL statement:
select o1_0."order_pk",o1_0."closed",o1_0."created",o1_0."customer_fk",o1_0."customer_uid",o1_0."order_total",o1_0."order_uid",o1_0."ready" from "orders" o1_0 where o1_0."order_uid"=? [42122-214]
2023-09-26 15:33:11 [http-nio-auto-1-exec-2] ERROR o.h.e.jdbc.spi.SqlExceptionHelper -
                Column "o1_0.customer_uid" not found; SQL statement:
select o1_0."order_pk",o1_0."closed",o1_0."created",o1_0."customer_fk",o1_0."customer_uid",o1_0."order_total",o1_0."order_uid",o1_0."ready" from "orders" o1_0 where o1_0."order_uid"=? [42122-214]
2023-09-26 15:37:20 [http-nio-auto-1-exec-2] ERROR o.h.e.jdbc.spi.SqlExceptionHelper -
                Column "o1_0.customer_uid" not found; SQL statement:
select o1_0."order_pk",o1_0."closed",o1_0."created",o1_0."customer_fk",o1_0."customer_uid",o1_0."order_total",o1_0."order_uid",o1_0."ready" from "orders" o1_0 where o1_0."order_uid"=? [42122-214]
2023-09-26 15:39:37 [http-nio-auto-1-exec-2] ERROR o.h.e.jdbc.spi.SqlExceptionHelper -
                Column "o1_0.customer_uid" not found; SQL statement:
select o1_0."order_pk",o1_0."closed",o1_0."created",o1_0."customer_fk",o1_0."customer_uid",o1_0."order_total",o1_0."order_uid",o1_0."ready" from "orders" o1_0 where o1_0."order_uid"=? [42122-214]
2023-09-26 15:42:23 [http-nio-auto-1-exec-2] ERROR o.h.e.jdbc.spi.SqlExceptionHelper -
                Column "o1_0.customer_uid" not found; SQL statement:
select o1_0."order_pk",o1_0."closed",o1_0."created",o1_0."customer_fk",o1_0."customer_uid",o1_0."order_total",o1_0."order_uid",o1_0."ready" from "orders" o1_0 where o1_0."order_uid"=? [42122-214]
2023-09-26 15:44:18 [http-nio-auto-1-exec-2] ERROR o.h.e.jdbc.spi.SqlExceptionHelper -
                Column "o1_0.customer_uid" not found; SQL statement:
select o1_0."order_pk",o1_0."closed",o1_0."created",o1_0."customer_fk",o1_0."customer_uid",o1_0."order_total",o1_0."order_uid",o1_0."ready" from "orders" o1_0 where o1_0."order_uid"=? [42122-214]
2023-09-26 15:58:47 [http-nio-auto-1-exec-2] ERROR o.h.e.jdbc.spi.SqlExceptionHelper -
                Column "o1_0.customer_uid" not found; SQL statement:
select o1_0."order_pk",o1_0."closed",o1_0."created",o1_0."customer_fk",o1_0."customer_uid",o1_0."order_total",o1_0."order_uid",o1_0."ready" from "orders" o1_0 where o1_0."order_uid"=? [42122-214]
2023-09-26 16:00:02 [http-nio-auto-1-exec-2] ERROR o.h.e.jdbc.spi.SqlExceptionHelper -
                Column "o1_0.customer_uid" not found; SQL statement:
select o1_0."order_pk",o1_0."closed",o1_0."created",o1_0."customer_fk",o1_0."customer_uid",o1_0."order_total",o1_0."order_uid",o1_0."ready" from "orders" o1_0 where o1_0."order_uid"=? [42122-214]
2023-09-26 16:02:23 [http-nio-auto-1-exec-1] ERROR o.h.e.jdbc.spi.SqlExceptionHelper -
                Column "o1_0.customer_uid" not found; SQL statement:
select o1_0."order_pk",o1_0."closed",o1_0."created",o1_0."customer_fk",o1_0."customer_uid",o1_0."order_total",o1_0."order_uid",o1_0."ready" from "orders" o1_0 where o1_0."order_uid"=? [42122-214]
2023-09-26 16:13:31 [http-nio-auto-1-exec-2] ERROR o.h.e.jdbc.spi.SqlExceptionHelper -
                NULL not allowed for column "customer_uid"; SQL statement:
insert into "orders" ("order_pk", "closed", "created", "customer_fk", "customer_uid", "order_total", "order_uid", "ready") values (default, ?, ?, ?, ?, ?, ?, ?) [23502-214]
2023-09-26 21:20:09 [http-nio-8080-exec-2] ERROR o.h.e.jdbc.spi.SqlExceptionHelper -
                Table 'tonsOfTacos.owners' doesn't exist
2023-09-26 21:52:29 [main] ERROR o.h.e.jdbc.spi.SqlExceptionHelper -
                Table "owners" not found (this database is empty); SQL statement:
select o1_0."owners_pk",o1_0."contact",o1_0."name",o1_0."psswrd",o1_0."role",o1_0."username" from "owners" o1_0 where o1_0."username"=? [42104-214]
2023-09-26 22:10:38 [main] ERROR o.s.test.context.TestContextManager -
                Caught exception while allowing TestExecutionListener [org.springframework.test.context.support.DependencyInjectionTestExecutionListener] to prepare test instance [com.adamstraub.tonsoftacos.tonsoftacos.mockTests.OrdersDbTests.NonEmptyTests@9b84747]
org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'com.adamstraub.tonsoftacos.tonsoftacos.mockTests.OrdersDbTests.NonEmptyTests': Unsatisfied dependency expressed through field 'mockMvc': No qualifying bean of type 'org.springframework.test.web.servlet.MockMvc' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.resolveFieldValue(AutowiredAnnotationBeanPostProcessor.java:712)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:692)
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:127)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessProperties(AutowiredAnnotationBeanPostProcessor.java:481)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1397)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.autowireBeanProperties(AbstractAutowireCapableBeanFactory.java:397)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:142)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:97)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:241)
	at org.springframework.test.context.junit.jupiter.SpringExtension.postProcessTestInstance(SpringExtension.java:138)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$invokeTestInstancePostProcessors$10(ClassBasedTestDescriptor.java:377)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.executeAndMaskThrowable(ClassBasedTestDescriptor.java:382)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$invokeTestInstancePostProcessors$11(ClassBasedTestDescriptor.java:377)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:179)
	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1625)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.StreamSpliterators$WrappingSpliterator.forEachRemaining(StreamSpliterators.java:310)
	at java.base/java.util.stream.Streams$ConcatSpliterator.forEachRemaining(Streams.java:735)
	at java.base/java.util.stream.Streams$ConcatSpliterator.forEachRemaining(Streams.java:734)
	at java.base/java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:762)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.invokeTestInstancePostProcessors(ClassBasedTestDescriptor.java:376)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$instantiateAndPostProcessTestInstance$6(ClassBasedTestDescriptor.java:289)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.instantiateAndPostProcessTestInstance(ClassBasedTestDescriptor.java:288)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$testInstancesProvider$4(ClassBasedTestDescriptor.java:278)
	at java.base/java.util.Optional.orElseGet(Optional.java:364)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$testInstancesProvider$5(ClassBasedTestDescriptor.java:277)
	at org.junit.jupiter.engine.execution.TestInstancesProvider.getTestInstances(TestInstancesProvider.java:31)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$prepare$0(TestMethodTestDescriptor.java:105)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.prepare(TestMethodTestDescriptor.java:104)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.prepare(TestMethodTestDescriptor.java:68)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$prepare$2(NodeTestTask.java:123)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.prepare(NodeTestTask.java:123)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:90)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:147)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:127)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:90)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:55)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:102)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:54)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:114)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:86)
	at org.junit.platform.launcher.core.DefaultLauncherSession$DelegatingLauncher.execute(DefaultLauncherSession.java:86)
	at org.junit.platform.launcher.core.SessionPerRequestLauncher.execute(SessionPerRequestLauncher.java:53)
	at com.intellij.junit5.JUnit5IdeaTestRunner.startRunnerWithArgs(JUnit5IdeaTestRunner.java:57)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:232)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:55)
Caused by: org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type 'org.springframework.test.web.servlet.MockMvc' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.raiseNoMatchingBeanFound(DefaultListableBeanFactory.java:1812)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1371)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1325)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.resolveFieldValue(AutowiredAnnotationBeanPostProcessor.java:709)
	... 76 common frames omitted
